### ISSUE TITLE
How  to train Face Expression Recognition?

### ISSUE THREAD

**Issue by DWCTOD** [sajjjadayobi/FaceLib#4]
Hi, hangsome boy, thanks for your nice work ,  could you share me how to train Facial Expression Recognition.  I didn't find something about these( Dataset and train code).
Hope your reply , thanks.

**Comment by sajjjadayobi** [comment]
it's UTKFace DataSet and trained with mobile net and resnet34 as the backbone and 
you can train with other backbones in FacialExpression/models/resnet.py
it's a classification and the loss is cross-entropy

**Comment by DWCTOD** [comment]
> it's UTKFace DataSet and trained with mobile net and resnet34 as the backbone and
> you can train with other backbones in FacialExpression/models/resnet.py
> it's a classification and the loss is cross-entropy

Thank you very much
